improver/nowcasting/optical_flow.py

```diff
@@ -88,7 +89,8 @@ def _calculate_time_average(wind_cubes, time_coord):
         """Average input cubelist over time"""
         cube = wind_cubes.merge_cube()
         try:
-            mean = cube.collapsed("time", iris.analysis.MEAN)
+
```

Comment:
Once again you have placed a rift between us. Why oh why?

Reply:
Because a regex went wild.